velox/exec/tests/utils/QueryAssertions.cpp

```diff
@@ -349,6 +349,15 @@ velox::variant arrayVariantAt(
   return velox::variant::array(std::move(array));
 }
 
+namespace {
+velox::variant const getNullDecimalVariant(::duckdb::Value& value) {
```

Comment:
This logic already exits elsewhere in this file and needs to be applies is a few more places. See https://github.com/facebookincubator/velox/compare/main...mbasmanova:velox-1:null_decimal?expand=1

Reply:
Updated, thanks!